client/components/user/index.jsx

```diff
@@ -6,7 +6,7 @@ var React = require( 'react' );
 /**
  * Internal dependencies
  */
-var Gravatar = require( 'components/gravatar' );
+var Gravatar = require( 'components/gravatar' ).default;
```

Comment:
Let's change this to an import usage. This protects us against breakages if `components/gravatar` goes back to exporting 1 thing.